README.md

```diff
@@ -569,7 +569,7 @@ These colorschemes may not specialize in Tree-sitter directly but are written in
 - [yaocccc/nvim-foldsign](https://github.com/yaocccc/nvim-foldsign) - Display folds on sign column.
 - [tenxsoydev/nx.nvim](https://github.com/tenxsoydev/nx.nvim) - Neovim API utility wrapper for more convenience with Lua keymaps, highlights, autocommands and options.
 - [zdcthomas/yop.nvim](https://github.com/zdcthomas/yop.nvim) - Easily create your own operators (like `d` and `y`).
-
+- [cpea2506/relative-toggle.nvim](https://github.com/cpea2506/relative-toggle.nvim) - A highly customizable plugin to automatically toggle smoothly between number and relative numbers, supporting various number combinations.
```

Comment:
The sentence probably has more punch with the shorter "Toggles smoothly between number and relative numbers, supporting various number combinations."

Shortening probably better conforms to the guideline about not mentioning the word 'plugin'

Reply:
Thank you for your recommendation! 😄